### DISCUSSION TITLE
Add an error message when trying to open a password protected album with the wrong password

### DISCUSSION THREAD

**Discussion by Gendra13** [LycheeOrg/Lychee#2762] [Ideas]
### Enhancement

Currently when trying to open a password protected album with the wrong password (mistyped), the prompt falls back to the default login dialog.

I believe it would be nice, if instead the password prompt keeps being display with a message like "wrong password" to give the user a second chance to enter the correct password.

**Comment by ildyria** [comment]
That is voluntary. We assume that you do not know the password, so we present you with a login dialog instead.

But maybe a better user experience would be to add a third button "Open Login" instead.
